arnold/__init__.py

```diff
@@ -1,164 +1,242 @@
 import os
+import sys
+
+import argparse
 
 from termcolor import colored
 
-from arnold.exceptions import (
-    ArgumentException,
-    DBAttrNotFound,
-    DirectionNotFoundException,
-    InvalidConfiguration,
-    ModuleNotFoundException,
-    MigrationNotFoundException,
-)
+from arnold.exceptions import DirectionNotFoundException
 from arnold.models import Migration
 from importlib import import_module
 
-import logging
-logger = logging.getLogger(__name__)
-
-
-IGNORED_FILES = ["__init__"]
 
+class Terminator:
+    IGNORED_FILES = ["__init__"]
 
-def _setup_table(model):
-    if model.table_exists():
-        return
-    else:
-        model.create_table()
-    return True
+    def __init__(self, args):
+        self.fake = getattr(args, 'fake', False)
+        self.count = getattr(args, 'count', 0)
+        self.folder = getattr(args, 'folder', None)
 
+        self.prepare_config()
+        self.database = self.config.database
```

Comment:
I wanted to keep this conifgurable. You are able to specify the database inside the config/__init__.py file.